classes/class-sailthru-horizon.php

```diff
@@ -319,7 +319,7 @@ public function register_plugin_scripts() {
 						wp_enqueue_script( 'personalize_js', '//ak.sail-horizon.com/spm/spm.v1.min.js' );
 						wp_register_script( 'tag', plugin_dir_url( __DIR__ ) . 'js/tag.js', array('jquery') );
 						wp_localize_script( 'tag', 'tag', $params );
-						wp_enqueue_script( 'tag' );
+						wp_enqueue_script( 'tag' , '', [], false, $this->js_tag_in_footer());
```

Comment:
Nitpicky, but the `tag` here I found confusing. Just switch to `js_in_footer` or simpler `in_footer()`. Simpler the better.